apps/token-wrapper/readme.md

```diff
@@ -11,3 +11,44 @@ The `TokenWrapper` contract has not yet been professionally audited. It is relat
 ## Caveats
 
 In efforts to save gas costs and space to introduce the checkpointing, token amounts are limited to `uint192`. This should not pose a problem for any token, but as `uint192` supports a _very_ large range of numbers, but the TokenWrapper will stop accepting deposits once if it hits `2^192 - 1`.
+
+## How to install
+
+**Prerequisites:** [aragonCLI](https://hack.aragon.org/docs/cli-intro) is needed to install the TokenWrapper to a DAO. Also note that you may have to change `--env aragon:rinkeby`  for  `--env aragon:mainnet` in case the organization is on mainnet.
```

Comment:
```suggestion
**Prerequisites:** [aragonCLI](https://hack.aragon.org/docs/cli-intro) is needed to install the Token Wrapper to a DAO. Also note that you may have to change `--env aragon:rinkeby` for `--env aragon:mainnet` in case the organization is on mainnet.
```